diff --git a/dist/index.cjs b/dist/index.cjs
index c7a9a0635880e92239067e985e3d03a73c2f3281..857c9b6d90f77d54ff48ab1c57e01238e3791f4a 100644
--- a/dist/index.cjs
+++ b/dist/index.cjs
@@ -340,9 +340,7 @@ function isOrderedList(line, cx, breaking) {
     }
     if (pos == line.pos || pos > line.pos + 9 ||
         (next != 46 && next != 41 /* '.)' */) ||
-        (pos < line.text.length - 1 && !space(line.text.charCodeAt(pos + 1))) ||
-        breaking && !inList(cx, Type.OrderedList) &&
-            (line.skipSpace(pos + 1) == line.text.length || pos > line.pos + 1 || line.next != 49 /* '1' */))
+        (pos < line.text.length - 1 && !space(line.text.charCodeAt(pos + 1))))
         return -1;
     return pos + 1 - line.pos;
 }
diff --git a/dist/index.js b/dist/index.js
index 07ce727452fe191aba6d5b3090c4585dd2135853..7374474e2db67da25f9e1089419a884a8a40516e 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -338,9 +338,7 @@ function isOrderedList(line, cx, breaking) {
     }
     if (pos == line.pos || pos > line.pos + 9 ||
         (next != 46 && next != 41 /* '.)' */) ||
-        (pos < line.text.length - 1 && !space(line.text.charCodeAt(pos + 1))) ||
-        breaking && !inList(cx, Type.OrderedList) &&
-            (line.skipSpace(pos + 1) == line.text.length || pos > line.pos + 1 || line.next != 49 /* '1' */))
+        (pos < line.text.length - 1 && !space(line.text.charCodeAt(pos + 1))))
         return -1;
     return pos + 1 - line.pos;
 }
